### ISSUE TITLE
Find a way to update META6.json with the provides field

### ISSUE THREAD

**Issue by FCO** [FCO/6pm#14]

**Comment by m-dango** [comment]
I wonder if it would be worth updating https://github.com/jonathanstowe/META6 with `scripts` as a customary field, and then using it in this module? I think it can be done on that one via [AT-KEY](https://docs.perl6.org/language/subscripts#method_AT-KEY).

e.g. `.AT-KEY("provides")<module> = 'filename';`

**Comment by FCO** [comment]
Yes, maybe augmenting it... I don’t think scripts should exist on META6 lib other then in 6pm... about how to add the provides, yes... but that’s the easiest part... I meant how should we discover what to add on that...

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by FCO** [comment]
Or if we should have a `6pm add-provider` that would create the file and add it on provides...

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
